src/adapters/gmx/products/glp/gmxGlpAdapter.ts

```diff
@@ -373,4 +359,76 @@ export class GMXGlpAdapter implements IProtocolAdapter, IMetadataBuilder {
   async getRewardApy(_input: GetApyInput): Promise<ProtocolTokenApy> {
     throw new NotImplementedError()
   }
+
+  async getDeposits({
+    userAddress,
+    protocolTokenAddress,
+    fromBlock,
+    toBlock,
+  }: GetEventsInput): Promise<MovementsByBlock[]> {
+    return await this.getMovements({
+      protocolToken: await this.fetchProtocolTokenMetadata(
+        protocolTokenAddress,
+      ),
+      underlyingTokens: await this.fetchUnderlyingTokensMetadata(
+        protocolTokenAddress,
+      ),
+      filter: {
+        smartContractAddress: protocolTokenAddress,
+        fromBlock,
+        toBlock,
+        from: ZERO_ADDRESS,
```

Comment:
In this case, it needs to be from `ZERO_ADDRESS`, not `undefined`.

Otherwise, given how the GLP token is deposited, it'll count both as a deposit and a withdrawal.

https://arbiscan.io/tx/0x47fbf2b3e9fc2c56d266469cb39f25b5e1404d96ad2d2a54d9f3838cb834e095